var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -169,6 +173,11 @@ def configure(self):
     def configure(self):
         configure('-fast',
                   '-no-webkit',
+                  # XXX(osx): sdk stuff; should depend on the architecture and the SDKs available.
+                  '-arch', 'x86_64',
+                  '-sdk', '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk',
```

Comment:
That's all fine, but I've had issues getting Qt4 to build with anything newer than the 10.9 SDK (which is no longer shipped). Do we just fail if a 10.9 SDK is missing?